tools/install-powershell.ps1

```diff
@@ -121,15 +148,19 @@ try {
         $packagePath = Join-Path -Path $tempDir -ChildPath $packageName
         Invoke-WebRequest -Uri $downloadURL -OutFile $packagePath
 
+        New-Item -ItemType Directory -Path "$Destination.new" > $null
         if ($IsWinEnv) {
-            Expand-Archive -Path $packagePath -DestinationPath $Destination
+            Expand-Archive -Path $packagePath -DestinationPath "$Destination.new"
         } else {
-            tar zxf $packagePath -C $Destination
+            tar zxf $packagePath -C "$Destination.new"
         }
+
+        Remove-Destination $Destination
+        Move-Item -Path "$Destination.new" -Destination $Destination
```

Comment:
If you can use `Move-Item` here, I think you can do the same in the `daily` code path without having to differentiate between `$Destination` exists or not (line 127).

Reply:
The differentiation is needed when `$Destination` is in use (because `install-powershell` was run from `$Destination`) so I can't use the `Move-Item` optimization.

Reply:
When installing a release package here, the `Destination` could also be `$PSHOME`, right?

Reply:
Yes, that's true.  Will fix.

Reply:
I just tried `install-powershell.ps1 -Destination $PSHOME` on Windows, and new files are not moved to $PSHOME after the run.